packages/ui/src/components/organisms/Form/DynamicForm/providers/ValidatorWrapper/ValidatorWrapper.tsx

```
@@ -0,0 +1,46 @@
+          type: document.type,
+          category: document.category,
+          entityType: document.businessId ? 'business' : 'ubo',
+          entityId: document.businessId ? document.businessId : document.endUserId!,
+        });
+
+        if (!document.files?.[0]) {
+          return acc;
+        }
+
+        return {
+          ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)